Gemfile.lock

```diff
@@ -127,4 +127,4 @@ DEPENDENCIES
   sqlite3 (~> 1.3)
 
 BUNDLED WITH
-   1.16.3
+   1.17.1
```

Comment:
nit: I think this code should be changed as well.

https://github.com/increments/graphql-kaminari_connection/blob/7a90ab17d6811f3b71f69b8f1655550691bd73ce/Gemfile.lock#L118

Reply:
I fix it